config/default/common/config/wv.json/layers/reference/Coastlines_15m.json

```diff
@@ -1,28 +1,32 @@
 {
   "layers": {
-    "Coastlines": {
-      "id": "Coastlines",
+    "Coastlines_15m": {
+      "id": "Coastlines_15m",
       "title": "Coastlines",
-      "description": "reference/Coastlines",
+      "description": "reference/Coastlines_15",
       "group": "overlays",
       "format": "image/png",
       "layergroup": "Reference",
       "noTransition": true,
       "projections": {
         "antarctic": {
+          "id": "Coastlines",
           "subtitle": "SCAR Antarctic Digital Database / Coastlines",
           "tags": "borders reference",
-          "source": "GIBS:antarctic"
+          "source": "GIBS:antarctic",
+          "matrixSet": "250m"
         },
         "geographic": {
           "subtitle": "&copy; OpenStreetMap contributors",
           "tags": "borders reference osm",
           "source": "GIBS:geographic"
         },
         "arctic": {
+          "id": "Coastlines",
```

Comment:
@edplato I went with `id` instead of `alt_id` because it was easier. In `layerBuilder.js` a layer's projection object is merged (similar to Object.Assign) with the main layer object. So, keeping it as `id` meant I didn't have to add new code.